### PR TITLE
adding filtered values to title on downloaded image

### PR DESCRIPTION
**Pull request by emre2038**
## Description
modifying the chart title on the downloaded image as `${title} by ${filterGroup}: ${filterValue}`

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/201

## How to test it locally
* run the project
* on the rich data panel, filter a chart
* download it as an image using the top-right menu
* check if the chart title has the correct format

## Screenshots
image

### Updated
* `horizontal-bar-chart.js` to get the chart title as a parameter and set it on `onClone` event of `html2canvas`
* `chart.js` to create the chart title string and send it as a parameter

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally
- [x] 👩‍🎨 does the design matches the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out
- [x] commit messages are meaningful

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

**Comment by emre2038** [comment]
@milafrerichs @adieyal there is an issue here. `:` is not allowed in file names on Windows. so when I download the image `:` is converted into `_`

**Comment by milafrerichs** [comment]
@emre2038 you misread the issue 😄 
It's about changing the title of the chart, not the filename.

**Comment by emre2038** [comment]
updated the PR @milafrerichs

File: src/js/profile/chart.js [hunk 1]
```diff
@@ -123,7 +123,9 @@ export class Chart extends Observable {
 
         $(saveImgButton).off('click');
         $(saveImgButton).on('click', () => {
-            barChart.saveAsPng(this.container);
+            let chartTitle = self.selectedGroup === null ? `${self.title}` : `${self.title} by ${self.selectedGroup} : ${self.selectedFilter}`;
```
[the remark this answers is not in the document]
Comment (milafrerichs) [reply src/js/profile/chart.js:126]: @emre2038 as codeclimate pointed out the same code is found twice. can you refactor this into a method of chart?

[the remark this answers is not in the document]
Comment (emre2038) [reply src/js/profile/chart.js]: @milafrerichs done

**Review by milafrerichs — APPROVED** [review]

**Pull request merged**